### ISSUE TITLE
Training

### ISSUE THREAD

**Issue by rebotnix** [ee09115/spoofing_detection#2]
Hi,

i would like to produce some own datasets with webcam and own print magazins. 

Do you can show us how you train the models by providing the training script?

Thx.

**Comment by ee09115** [comment]
Hello,

I have added the script I used to train the ETC classifier. 

The input file is a csv file which have the following format:

`<feature vector 1>,<label 1>,<PAI>`
`<feature vector 2>,<label 2>,<PAI>`
`...`
`<feature vector N>,<label N>,<PAI>`

 The feature vector is a floating point vector, FV=(Y,Cr,Cb,Luv), which contain the frequency histogram normalized in the range [0.0, 255.0]. 
The label is the corresponding label to the nature of the feature vector: I used 1 for an attack, and 0 for a bona fide image.
The PAI, is the Presentation Attack Instrument used to produce the attack. It is useful when you want to use ACER metric to evaluate your PAD instrument.

Regards,
